### PR TITLE
added b-products, b-items, b-news, b-newsletter, b-footer section - html+css

### PR DESCRIPTION
**Pull request by karokaro**
I've almost finished coding the project. Please do cr and leave me some tips. The next step is adding some js. Thanks in advance.

### CODE REVIEW

File: kd/brandly/common/css/style.css [hunk 1]
```diff
@@ -86,21 +86,71 @@ body {
...
+  font: 900 20px "Lato", sans-serif; }
+@media (min-width: 700px) {
+  body h4 {
+    font-size: 30px; } }
 body .m-wrapper {
-  width: 1170px;
   margin: 0 auto;
-  padding: 20px 20px 0; }
-@media (min-width: 700px) {
+  padding: 0 20px 0; }
```
Comment (pdrazewski) [inline kd/brandly/common/css/style.css]: you should change breakpoint value for this part to be equal with breakpoint value for "body .b-products .m-wrapper"
right now you've got 100% + 40px width for ".b-products .m-wrapper" - witch result with having horizontal scroll bar appearing at 1366px

**Comment by pdrazewski** [comment]
you should change breakpoint value for "body .m-wrapper" to be equal with breakpoint value for "body .b-products .m-wrapper".

Right now you've got 100% + 40px width for ".b-products .m-wrapper" - witch result with having horizontal scroll bar appearing at 1366px

**Comment by pdrazewski** [comment]
for 320px screen size you need to work on:
- spacing for headlines (lack of symetry)
- spacing in newsletter box

**Comment by pdrazewski** [comment]
"subscribe" button should have cursor pointer

**Comment by pdrazewski** [comment]
describe what you mean by using b-, m-, el- prefixes in your class names

**Comment by pdrazewski** [comment]
"latest products" looks bad beetwen 500 and 850px

**Comment by karokaro** [comment]
b-
it's the prefix for blocks. The most general container eg. news, products, hero panel

m- 
it's the prefix for modules. They are inside block, you can have different module on page: module which will contain categories, promotion, headline, pagination, slider etc. It's more specyfic than blocks but not as detailed as element

el-
it's the prefix for one element. It's the smallest element of code (little part of one tag) It has really narrow meaning

**Comment by karokaro** [comment]
Fixes added. Please check.

**Pull request merged**